### PR TITLE
sprint#16/AppStore-version-override

### PR DESCRIPTION
**Pull request by ReggaePanda**
- add version override
- add changlog plugin (changeling changed)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -112,16 +112,18 @@ matrix:
...
+    - openssl aes-256-cbc -k "$SECRET" -in .env.production.enc -out .env.production -d
+    - openssl aes-256-cbc -k "$SECRET" -in android/Google_Play_Android_Developer.json.enc -out android/Google_Play_Android_Developer.json -d
+    - openssl aes-256-cbc -k "$SECRET" -in android/app/democracy2-release-key.keystore.enc -out android/app/democracy2-release-key.keystore -d  
+    - cd android
     - gem update --system
     - bundle install
     - bundle update fastlane
     install:
-    - cd client
+    - cd ..
```
Comment (ulfgebhardt) [inline .travis.yml]: Is this correct?

File: .travis.yml [hunk 2]
```diff
@@ -233,13 +239,15 @@ matrix:
...
+    - cd client
+    - openssl aes-256-cbc -k "$SECRET" -in .env.alpha.enc -out .env.alpha -d
+    - openssl aes-256-cbc -k "$SECRET" -in .env.beta.enc -out .env.beta -d
+    - openssl aes-256-cbc -k "$SECRET" -in .env.production.enc -out .env.production -d
+    - cd ios
     - bundle install
     - bundle update fastlane
     install:
-    - cd client
+    - cd ..
```
Comment (ulfgebhardt) [inline .travis.yml]: Is this correct?

File: .travis.yml (hunk 1 above)
Comment (ReggaePanda) [reply .travis.yml]: yes, the travis `grandfile` parameter does not work. so the script have to change the dir to install all correct

File: .travis.yml (hunk 2 above)
Comment (ReggaePanda) [reply .travis.yml]: yes, the travis `grandfile` parameter does not work. so the script have to change the dir to install all correct

**Pull request merged**